src/components/VStep.vue

```diff
@@ -8,7 +8,10 @@
 
     <slot name="content">
       <div class="v-step__content">
-        <div v-if="step.content" v-html="step.content"></div>
+        <template v-if="step.content">
+          <div v-if="typeof step.content === 'string'" v-html="step.content"></div>
```

Comment:
How about if it's an array or an object? The v-component will more than likely error when passing those to the `is` prop

Reply:
Indeed, it may be better to check if the variable is a Vue component by using `v-if="step.content instanceof Vue"` on v-component and throw everything else in the v-html attribute.

Reply:
I think I tried checking as you say @mmorainville, but I think that didnt work for some reason :(